manifests/README.md

```diff
@@ -0,0 +1,134 @@
+# Manifests
+
+This directory contains the manifests for deploying Boots to various environments.
```

Comment:
Would it makes sense to have a quick compare&contrast of these _ as they relate to boots_ (not in general). For example you mention

> **Note:** KinD will not be able to listen for DHCP broadcast traffic. Using a DHCP relay is recommended.

Does that apply to any other setups?

Reply:
sure, i'll add something.

Reply:
This note only applies to KinD.
Also, I've updated the readme with a visual comparison. Please check it out when you have a chance.

Reply:
:+1: on the image (I wondered if it could be recreated in mermaidjs to avoid a binary file, but I didn't really see anything as nice as the image you have). I'm a little confused by the visualization. Does linux need both k3d and kind?

Reply:
> Does linux need both k3d and kind?

No, the diagram is a comparison only.

Reply:
gotcha, maybe have them at the same "level" and avoid the arrows making it look like traffic goes from one to the other?

Reply:
hmmm.....I could move the boxes around, but the arrow is coming out of the host network ns not the K3D container. the arrows purpose is to illustrate the traffic flow to the KinD container.

Reply:
I guess I'm just confused here because it looks like the image is saying we're running 2 boots containers. One in K3D and one in KinD and the one in K3D talks to KinD for some reason? I like the images, but I don't want to add any confusion if someone is looking at them. Can you do one image for K3D and one for KinD? Or some better separation so that the image isn't implying that they are communicating to each other. I think part of the confusion is also that the 2 different setups have different communication flows, why wouldn't k3D and kind both have similar flows (note I don't mean where the arrows originate/terminate but the direction K3D has outgoing arrows, KinD has incoming).